test/get-module-specifier-test.js

```diff
@@ -51,90 +51,51 @@ describe('get-module-specifier', function() {
     moduleConfig = getModuleConfig(resolverConfig);
   });
 
-  it('identifies named modules in the root of a collection as the default type for that collection', function() {
```

Comment:
This test still seems valid to me. Any reason to remove it instead of update for the new signature of `getModuleSpecifier`?